sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/Common/AbstractExplodedGraph.cs

```diff
@@ -32,7 +32,7 @@ namespace SonarAnalyzer.SymbolicExecution
 {
     internal abstract class AbstractExplodedGraph
     {
-        internal const int MaxStepCount = 1000;
+        internal const int MaxStepCount = 1500;
```

Comment:
Had to increase the step count to avoid missing TP which we raised in previous versions. We will lose a bit of the performance improvement here but we will get more issues reported.